lib/Agrammon/Web/Routes.pm6

```diff
@@ -268,6 +268,19 @@ sub api-routes (Str $schema, $ws) {
                 }
             }
         }
+        operation 'getExcelExport', -> LoggedIn $user {
+            request-body -> ( :%params ) {
+                my $export = $ws.get-excel-export($user, %params);
+                # change to application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
+                # once Spreadsheet::XLSX is working again
+                content 'text/csv', { $export };
```

Comment:
I don't think you want a hash here (in fact, it's probably an error to have one). You just want the string `$export`.

Reply:
Fixed.